src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -284,6 +285,95 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
     }
     
+    private void testEditVisibilityAction() throws Exception {
+
+        ______TS("edit question 1 to Giver's team members and Giver");
```

Comment:
If I use xpath to locate this row, I cannot find the row "Giver's Team Members and Giver ". If I use css selector findElements(By.cssSelector("hide[style*='none']") and try to get the content of this row, "org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with" is thrown.